.bazelci/presubmit.yml

```diff
@@ -12,3 +12,11 @@ tasks:
   windows:
     build_targets:
     - "//..."
+  ubuntu2004_last_green_bzlmod:
+    name: "Last Green Bazel (with bzlmod)"
+    platform: ubuntu2004
+    bazel: last_green
```

Comment:
Don't bother with bazel:last_green. We should be aligned with the bazel we use for the other tests.